pkgs/os-specific/linux/kernel/xanmod-kernels.nix

```diff
@@ -6,8 +6,8 @@ let
   # NOTE: When updating these, please also take a look at the changes done to
   # kernel config in the xanmod version commit
   ltsVariant = {
-    version = "6.6.51";
-    hash = "sha256-dNUTePfL6cAA0EmEG/D36dNJUobDRBHR5+BYrLIYot4=";
+    version = "6.11.2";
+    hash = "sha256-4BXPZs8lp/O/JGWFIO/J1HyOjByaqWQ9O6/jx76TIDs=";
```

Comment:
That sounds wrong here.

Reply:
Indeed, I messed up the cherry-pick.

Thanks a bunch for catching my mistake, I really appreciate it!